frontend/simple/views/Mailbox.vue

```diff
@@ -0,0 +1,284 @@
+              <div class="panel-block">
+                <span class="panel-icon">
+                  <i class="fa fa-pencil"></i>
+                </span>
+                <a id="ComposeLink" v-on:click="compose"><i18n>compose</i18n></a>
+              </div>
+              <div class="panel-block">
+                <span class="panel-icon">
+                  <i class="fa fa-envelope"></i>
+                </span>
+                <a id="InboxLink" v-on:click="inboxMode"><i18n>inbox</i18n> <span class="unread" v-if="$store.getters.unreadMessageCount">{{ $store.getters.unreadMessageCount }}</span></a>
+              </div>
+            </div>
+          </div>
+          <div class="column"></div>
+          <div class="column is-two-thirds">
+            <div id="compose" class="panel" v-show="mode === 'Compose'">
+              <div class="panel-heading">
+                <div><strong><i18n>Type</i18n>:</strong>&nbsp;<i18n>Message</i18n></div>
+                <div><strong style="margin-top: auto"><i18n>To</i18n>:</strong>&nbsp;
+                  <input id="AddRecipient" class="input is-small" type="text" style="width: 80%;" v-model="recipient" v-on:blur="addRecipient">
+                  <a class="button is-small" v-on:click="addRecipient">
+                    <span class="icon is-small">
+                      <i class="fa fa-plus-circle"></i>
+                    </span>
+                  </a>
+                </div>
+                <span v-if="error" id="badUsername" class="help is-danger"><i18n>Invalid Username</i18n></span>
+                <table>
+                  <tr v-for="(recipient, index) in recipients">
+                    <td>
+                      {{recipient.name}}
+                      <i class="fa fa-minus-circle icon is-small" v-on:click="removeRecipient(index)" ></i>
+                    </td>
+                  </tr>
+                </table>
+              </div>
+              <div class="panel-block">
+                <textarea id="ComposedMessage" class="textarea" v-model="composedMessage"></textarea>
+              </div>
+              <div class="panel-block" >
+                <button id="SendButton" class="button is-success" type="submit" v-on:click="send" :disabled="!composedMessage"  style="margin-left:auto; margin-right: 0"><i18n>Send</i18n></button>
+                <button class="button is-danger" type="submit" v-on:click="cancel" style="margin-left:10px; margin-right: 0"><i18n>Cancel</i18n></button>
+              </div>
+            </div>
+            <div id="CurrentMessage" class="panel" v-show="mode === 'Read'">
+              <div class="panel-heading">
+                <div><strong>Type:</strong>&nbsp;{{ currentMessage.groupId ?  'Invite' : 'Message'}}</div>
+                <div><strong>Sent:</strong>&nbsp;{{formatDate(currentMessage.sentDate)}}</div>
+                <div><strong>From:</strong>&nbsp;{{ currentMessage.groupId ?  currentMessage.groupId : currentMessage.from}}</div>
```

Comment:
These `&nbsp;`'s appear to be here for no reason. If there's no reason for them, replace them with a normal space, otherwise the reason must be explained, as this is "code smell".